src/index.md

```
@@ -31,6 +31,24 @@ Software engineer based in Tokyo. 🇯🇵
 
 I'm a software engineer on the Wevox product team, participating in the entire process from organizing user requirements to product design and system development.
 
+:sparkles: Workflow :sparkles:
+
+```
```

Markdownlint

Fenced code blocks should have a language specified

(MD040, fenced-code-language)